main.cpp

```diff
@@ -99,53 +104,76 @@ struct Skateboard
 
 void Skateboard::rollDownStreet(int speed)
 {
-    
+  if (speed > 20)
+    {
+        std::cout << "Slow Down\n.";  
+
+    }
 
 }
 void Skateboard::railSlide(bool frontSide)
 {
+    if (frontSide)
+    {
+        std::cout << "Now try it backside, dude!\n.";
+    }
+    else
+        std::cout << "Now try it frontside, dude!\n.";
 
```

Comment:
Use curly braces for both `if()` and `else()` blocks, not for one but not the other.  it is hard to read. 
- [ ] revise.